src/pages/en/guides/imports.md

```diff
@@ -165,6 +165,23 @@ const posts = await Astro.glob('../pages/post/*.md');
 </div>
 ```
 
+Imported Astro components via `Astro.glob` are of type [`AstroInstance`](/en/reference/api-reference/#astro-files), so we can access our components under the `default` property.
```

Comment:
We usually use “You” rather than “We” when talking about what a user can do. A youser!

I’ve tried to rework this here:

```suggestion
Astro components imported using `Astro.glob` are of type [`AstroInstance`](/en/reference/api-reference/#astro-files). You can render each component instance using its `default` property:
```